xcp_d/workflow/anatomical.py

```diff
@@ -1,5 +1,5 @@
 # emacs: -*- mode: python; py-indent-offset: 4; indent-tabs-mode: nil -*-
-# vi: set ft=python sts=4 ts=4 sw=4 et:
+# vi: set ft=python sts=4 ts=4 sw=4 init_structral_wfbet:

```

Comment:
Is this a typo?

Reply:
yep, thanks for catching -- will fix that right now